module/common/ManagePwm.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file ManagePwm.h
+ * @brief 走行体の左右車輪のPWMを所持するクラス
+ * @author bizyutyu
+ */
+
+#ifndef MANAGEPWM_H
+#define MANAGEPWM_H
+
+#include <stdio.h>
+
+class ManagePwm {
+ public:
```

Comment:
```suggestion
 public:
  ManagePwm() = delete;  // 明示的にインスタンス化を禁止
```
ここで明示的にインスタンス化を禁止してほしい